alphastats/DataSet_Preprocess.py

```diff
@@ -1,14 +1,15 @@
-from random import random
-import pandas as pd
-import sklearn
+import itertools
```

Comment:
please don't mix formatting with any functional changes as it is sometimes hard to distinguish those changes from each other .. 
I you want, we could format the whole code base together one (cf. https://github.com/MannLabs/alphabase/issues/153), should take only 15 minutes ;-)

Reply:
Sounds good to me, when would you have the time?